src/main/java/net/dv8tion/jda/client/entities/impl/JDAClientImpl.java

```diff
@@ -191,4 +193,29 @@ public UserSettings getSettings()
     {
         return callUsers;
     }
+
+    @Override
+    public RestAction<Invite> acceptInvite(Invite invite)
```

Comment:
I've put this into `JDAClient` because it's client only and thus should go there alongside all other client only features. It would be good to mention it in the `Invite` class level docs though.